src/liveUpdates.ts

```diff
@@ -210,16 +210,66 @@ export class LiveUpdates {
     }
   }
 
+  /**
+   * **Basic** validating of the subscribed data
+   * Is it GraphQL or REST and does it contain the sys information
+   * TODO: add more accurate checks
+   */
+  private validateDataFromPreview(data: Argument) {
+    const dataAsString = JSON.stringify(data);
+
+    const isGQL = dataAsString.includes('__typename');
+    const isREST = dataAsString.includes('fields":{');
+    const hasSys = dataAsString.includes('sys":{');
+
+    let isValid = true;
+
+    if (!hasSys) {
+      isValid = false;
+      debug.error('Live Updates requires the "sys.id" to be present on the provided data', data);
+    }
+
+    if (!isGQL && !isREST) {
+      isValid = false;
+      debug.error(
+        'Live Updates requires the the provided data to contain either "fields" or "__typename"',
```

Comment:
I think we can be explicit here for REST and Graphql.

`For live updates as a basic requirement the provided data must include the "fields" property for REST or "__typename" for Graphql, otherwise the data will be treated as invalid and live updates will not work.`

what do you think about something like this?